content/guides/validator/introduction.md

```diff
@@ -232,26 +232,43 @@ You can also manually construct the class instance and pass any arguments you li
 ```ts
 Route.post('users', async ({ request, response }) => {
   await request.validate(
-    new CreateUser({
+    new CreateUserValidator({
       countries: fetchAllowedCountries(),
       states: fetchAllowedStates()
     })
   )
 })
 ```
 
-Following is an example of using the validator classes outside of the HTTP request.
+Your validator class would look like the following:
```

Comment:
The sentence must include `validator classes usage outside of the HTTP request`

Reply:
Changed the wording. Let me know if you need any changes to my modified wording.